numpy/testing/_private/utils.py

```diff
@@ -713,7 +713,7 @@ def func_assert_same_pos(x, y, func=isnan, hasval='nan'):
         # such subclasses, but some used to work.
         x_id = func(x)
         y_id = func(y)
-        if npall(x_id == y_id) != True:
+        if (x_id == y_id).all() != True:
```

Comment:
Given the comment above, I think this should not be changed... Alternatively, at least the comment should be adjusted.

Reply:
Good catch. The problem is that now `np.all()` may likely not be implemented for an ndarray subclass, if it doesn't define `np.all` in `__array_function__`.

I think we can just delete this comment, because it's no longer true that `np.all()` can work when a subclass implements`.all()` differently -- we already use `getattr()` to pull out a `all()` method to call if one exists.

Reply:
The reason I worry is that this change is a very recent one, by @charris (https://github.com/numpy/numpy/pull/11756), which correctly something that `.all()` didn't catch. (Sorry, have to run, not sure what the real issue was...)

Reply:
OK, I see, thanks. I think the comment was a little misleading here -- I think the problem was classes that define equality differently (e.g., to return a boolean) rather than classes that don't define an `.all()` method.